parm/atm/obs/lists/gdas_prototype_3d.yaml.j2

```diff
@@ -0,0 +1,8 @@
+observers:
+{% filter indent(width=2) %}
+{% include 'atm/obs/config/satwind_abi_goes-16.yaml.j2' %}
+{% include 'atm/obs/config/scatwind_ascat_metop-b.yaml.j2' %}
+{% include 'atm/obs/config/conv_ps.yaml.j2' %}
+{% include 'atm/obs/config/gnssro.yaml.j2' %}
+{% set cycle = current_cycle | strftime("t%Hz") %}
```

Comment:
```suggestion
```
Remove `cycle` as it is not used in the template.

Reply:
Change committed at b174bf2